ls1intum/Artemis
# Allow markdown in course registration confirmation message

### Checklist

### Motivation and Context
fixes #3216

### Steps for Testing


1. Navigate to Course Administration
2. Create or edit a course. Enable student registration and use the markdown editor to create a registration confirmation message and save the changes.
3. Log in with an account that can enroll for the course.
4. Check if previously created message is displayed as intended.

### Test Coverage




### Screenshots

https://user-images.githubusercontent.com/44805696/115958463-e62d1380-a507-11eb-9f86-328d263f90f6.mp4
